### PR TITLE
Enhance UI + Add token screen

### PR DESCRIPTION
**Pull request by lethuccrma**
- Enhance Login screen
- Add token screen
- Show added tokens

https://user-images.githubusercontent.com/105472435/177366077-a8bcc5c0-a4af-4ec2-a3fb-8e1f0f66608a.mp4

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ module.exports = {
     'react/jsx-filename-extension': [1, { extensions: ['.js', '.jsx'] }],
     'no-unused-vars': 'warn',
     'react/jsx-no-constructed-context-values': 'off',
+    'no-param-reassign': 'warn',
```
Comment (lytienphuc0810) [inline .eslintrc.js]: this should not be bypassed

File: .eslintrc.js [hunk 2]
```diff
@@ -5,6 +5,9 @@ module.exports = {
...
+    'react/prop-types': 'warn',
+    'import/no-cycle': 'warn',
```
Comment (lytienphuc0810) [inline .eslintrc.js]: this also should not be bypassed. Circular dependency is very bad

File: .eslintrc.js (hunk 1 above)
Comment (lytienphuc0810) [reply .eslintrc.js]: Params should be treated as const

**Review by lytienphuc0810 — CHANGES_REQUESTED** [review]

**Comment by lethuccrma** [comment]
Hi a @lytienphuc0810 . All issues have been fixed without bypassing eslint rules

**Review by lytienphuc0810 — APPROVED** [review]

**Pull request merged**